autoload/wiki/fzf.vim

```diff
@@ -0,0 +1,53 @@
+function! wiki#fzf#pages() abort
+  if !exists('*fzf#run')
+    echo "fzf must be installed for this to work"
+    return
+  endif
+  let l:root = wiki#get_root()
+
+  let l:pattern = '**/*.' . join(g:wiki_filetypes, ',')
+  let l:pages = split(globpath(l:root, l:pattern))
+  let l:pages = map(l:pages, '"/".substitute(v:val, l:root . "/" , "", "")')
+
+  call fzf#run({
+  \   'source': l:pages,
```

Comment:
Done

Reply:
Sorry, I realize I should have been more clear. The indents should be like this:

```
call fzf#run({
      \ 'source': l:pages,
      \ 'sink': funcref('s:pages_accept'),
      \ 'options': '--prompt "WikiPages> " '
      \})
```

That is, the `\`s should be indented. But I'll accept it as it is and fix it myself later, since I'm the one with the OCD...